modules/runtime/pages/process-manager-application.adoc

```diff
@@ -0,0 +1,60 @@
+= Creating a process manager application
+:description: This page explains how to create your own custom process manager application
+
+[NOTE]
+====
+For Subscription editions only. +
+If your Bonita Subscription has been purchased before 2021, this Add-on can be purchased on top of your subscription.
+If you have purchased or renewed your Bonita Subscription in 2021 or after, BCD is fully part of your subscription.
+====
+
+This page explains how to create your own custom process manager application. +
+This application functions like the Administrator application, but has a smaller scope, one that only controls processes, cases and tasks. +
+Creating such an application follows the same steps as the creation of any xref:applications:application-creation.adoc[other custom application].
+
+== Pages
+
+The main menu of the application will require four pages.
+
+* Monitoring
+* Process list
+* Case list
+* Task list
+
+In addition to those, there are some orphan pages that are required by the above ones :
+
+* Process details
+* Process visualization
+* Case details
+* Case visualization
+* Task details
+
+== Page modifications
+
+The above-mentioned menu pages are the same as the ones that are part of the Administrator application, but they only show the processes/cases/tasks that the user supervises. +
+In order to make the menu pages do so, you need to export them from the Administrator Resource List and edit them using the UI Designer.
+You don't need to export and modify the orphan pages.
+
+[WARNING]
+====
+
+Currently, the Monitoring page is made using a different technology than UI Designer, and thus it cannot be edited.
+====
+
+The modifications that you will need to do are quite simple:
+
+1. Though not necessary, I highly recommend re-saving your pages with a different name. This will prevent the pages from being over-written if you need to re-import the default pages.
```

Comment:
use the impersonal form as much as possible in documentation, especially avoid the 'I'. Users do not want 'your' advice but 'the' recommendation. 
```suggestion
1. Though not necessary, it is highly recommended to re-save pages with a different name. This will prevent these pages from being over-written if you need to re-import the default pages.
```